docs/B-Computations/logic.md

```diff
@@ -69,7 +73,7 @@ _**Pseudo-code**_ is a set of shorthand notes in a human \(non-programming\) lan
 
 A _**flow chart**_ is a set of conventional symbols connected by arrows that illustrate the flow of control through a programming solution.  Popular sets of symbols for sequences, selections and iterations are shown below:
 
-![](https://ict.senecacollege.ca//~ipc144/pages/images/flowChart.png)
+![flowChart](../../static/img/image40.png)
```

Comment:
Suggestion to provide meaningful alt text. Example: `Popular symbols for sequence, selection, and iteration constructs`